### DISCUSSION TITLE
Title含有空格导致自定义下载文件名问题

### DISCUSSION THREAD

**Discussion by BreakshadowCN** [the1812/Bilibili-Evolved#1689] [提问 / Q&A]
e.g.：https://www.bilibili.com/bangumi/play/ep350181

视频的Title可以是
'说唱新世代：Feezy 《新浪漫主义》'（h1Title，注意有空格）
也可以是
'Feezy 《新浪漫主义》'（toast_title，注意有空格）
但是目前脚本获取title采用的方式会获取到'说唱新世代：Feezy_综艺_bilibili_哔哩哔哩'，实际的歌名没了，这种情况会影响所下载视频的整理
这应该是b站自身的问题，但是希望这边能做些调整适配

除此之外，想下载上述视频，在文件命名格式只填入[ep]后下载，发现ep参数是空的（生成了叫xml.xml的弹幕文件）。不知道这是什么样的情况（这个视频怎么看都像是有选集标题这一说）。

**Comment by the1812** [comment]
不是因为有空格, 而是你选的是`纯享舞台`这一块的视频, 标题相关的功能只能处理上面的`正片`.

**Reply by BreakshadowCN** [reply]
> 不是因为有空格, 而是你选的是`纯享舞台`这一块的视频, 标题相关的功能只能处理上面的`正片`.

多谢回答。请问会考虑支持“正片”以外部分的处理吗？

**Reply by BreakshadowCN** [reply]
> 不是因为有空格, 而是你选的是`纯享舞台`这一块的视频, 标题相关的功能只能处理上面的`正片`.

用同样的视频做了试验，采用默认的[title][ - ep]作为标题的时候，得到的文件名仍然是“说唱新世代：Feezy”，歌名的部分没有办法获取到。请问怎么解决比较好？

**Reply by the1812** [reply]
目前只能手动复制歌名了
